### ISSUE TITLE
Jalali Calendar Support 

### ISSUE THREAD

**Issue by mmirzaeyan** [mattlewis92/angular-calendar#779]
**Is your feature request related to a problem? Please describe**
based on #715 i try to provide jalali adapter but i dont know how to change month and week days name , after you override the adapter the date converted to gregorian date.
if you want to i'm add this feature to this library please don't close it with post just a simple answer.

**Describe the solution you'd like**

**Describe your use case for implementing this feature**

**Additional context**

**Comment by mattlewis92** [comment]
Yes please, that would be awesome! You can [make a PR](http://makeapullrequest.com) to [add it as a demo](https://github.com/mattlewis92/angular-calendar/tree/master/projects/demos/app/demo-modules). Let me know if you need any more help 😄

**Comment by mattlewis92** [comment]
Or you could even publish `jalali-date-formatter.provider.ts` and `jalali-date-adapter.provider.ts` to their own npm package to make the code more reusable. You can use the [angular cli](https://github.com/angular/angular-cli/wiki/stories-create-library) to do this.

**Comment by mmirzaeyan** [comment]
is there any way to use 0.26 on angular 5 project ? i cant migrate to angular 6 with easily.

**Comment by mattlewis92** [comment]
You might be able to by installing the `rxjs-compat` package and updating to rxjs 6, that's where most of the breaking changes were between v5 and v6

**Comment by rahimzade** [comment]
hi
in[ this sample](https://github.com/mattlewis92/angular-calendar/issues/779#issuecomment-433612309) when  use JalaliAdapter in module like this : ` useFactory() {
        return new JalaliAdapter();
      }`  then `weekStartsOn: number = DAYS_OF_WEEK.SATURDAY;` not working!
is any way to resolve this problem?

**Comment by mattlewis92** [comment]
> hi
> in[ this sample](https://github.com/mattlewis92/angular-calendar/issues/779#issuecomment-433612309) when use JalaliAdapter in module like this : ` useFactory() { return new JalaliAdapter(); }` then `weekStartsOn: number = DAYS_OF_WEEK.SATURDAY;` not working!
> is any way to resolve this problem?

It's because moment doesn't support passing the day of the week to each function call, you have to configure it globally like this:
```
moment.updateLocale('fa', {
  week: {
    dow: DAYS_OF_WEEK.SATURDAY,
    doy: 0,
  },
});
```

Hope that helps! 😄

**Comment by arashtaj** [comment]
hi 
how can I fix events (drag and drop) for jalali calendar ...
when I want to drag it to other days it removed ...
how can I fix this problem .
Thanks .
